zdm/iteration.py

```diff
@@ -131,53 +132,66 @@ def get_log_likelihood(grid, s, norm=True, psnr=True, Pn=False, pNreps=True):
     if isinstance(grid, zdm_repeat_grid.repeat_Grid):
         # Repeaters
         if s.nDr==1:
-            llsum1, lllist, expected = calc_likelihoods_1D(grid, s, norm=norm, psnr=psnr, dolist=1, grid_type=1, Pn=Pn, pNreps=pNreps)
+            llsum1, lllist, expected = calc_likelihoods_1D(grid, s, norm=norm, psnr=psnr,
```

Comment:
this snarled code needs refactoring someday; but not today

Reply:
Yep... these functions are pretty long and painful. But... not today! :-)